versioneer.py

```diff
@@ -1719,7 +1790,7 @@ class VersioneerBadRootError(Exception):
     """The project root directory is unknown or missing key files."""
 
 
-def get_versions(verbose=False):
+def get_versions(verbose: bool = False) -> Dict[str, Any]:
     """Get the project version from whatever source is available.
```

Comment:
One-line sphinx docstring (missing the description of input parameters, errors raised, what's returned, etc.: :param :return)